watertap/unit_models/osmotically_assisted_reverse_osmosis_base.py

```diff
@@ -340,15 +343,23 @@ def _add_area(self, include_constraint=True):
             )
 
         if include_constraint:
-            if not hasattr(self, "eq_area"):
```

Comment:
Why delete this check to make sure `eq_area` isn't already defined on the model?

Reply:
I assume we originally had this here because there was potential for a user to erroneously replicate this constraint without realizing, which would cause issues with the Pyomo model.

Reply:
That might be an error caused by copy and paste. I had no reason to remove that but it's not in the RO_base, so likely I copied the structure over and erroneously removed that check

Reply:
Placed it back now